qiskit/circuit/library/pauli_evolution.py

```diff
@@ -85,10 +86,10 @@ class PauliEvolutionGate(Gate):
 
     def __init__(
         self,
-        operator,
-        time: Union[int, float, ParameterExpression] = 1.0,
```

Comment:
why is ParamterExpression replaced by ParameterValueType ? (isn't this considered as an API change) ?

Reply:
It's the same, `ParameterValueType` is a typedef used across the circuit and the gates, it's defined as 
```python
ParameterValueType = Union[ParameterExpression, float]
```
(see the `quantumcircuit.py` file)